<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KBugsContext</class>
 <widget class="QWidget" name="KBugsContext">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>KBugsContext</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/res/images/tray.ico</normaloff>:/images/res/images/tray.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="configFIlePathLabel">
         <property name="toolTip">
          <string>kubeconfig 文件或目录（*.config 文件将被加入）</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string>&amp;Kubeconfig File/Dir：</string>
         </property>
         <property name="buddy">
          <cstring>kubeConfigPathLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="kubeConfigPathLineEdit">
         <property name="placeholderText">
          <string>kubeconfig file/dir(only *.config will be added)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="changeKubeConfigDirBtn">
       <property name="text">
        <string>&amp;Open</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fromServerBtn">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>&amp;From Server</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>&amp;Name</string>
         </property>
         <property name="buddy">
          <cstring>nameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="clusterLabel">
         <property name="text">
          <string>&amp;Cluster</string>
         </property>
         <property name="buddy">
          <cstring>clusterCombox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="clusterCombox"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="clusterServerLabel">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Server:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="userLabel">
         <property name="text">
          <string>&amp;User</string>
         </property>
         <property name="buddy">
          <cstring>userComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="userComboBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="nameSpaceLabel">
         <property name="text">
          <string>Name&amp;Space</string>
         </property>
         <property name="buddy">
          <cstring>nameSpaceLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="nameSpaceLineEdit"/>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="setColorCheckBox">
         <property name="text">
          <string>Set Icon color for this context</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QListView" name="ctxListView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DropOnly</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="toggleCtxCheckBox">
         <property name="text">
          <string>Show current context in menu bar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="revertCtxConfigBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Revert</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+Z</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="applyCtxConfigBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Apply</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>canRevert(QString)</slot>
 </slots>
</ui>
